src/prettytable/prettytable.py

```diff
@@ -2214,11 +2253,16 @@ def _get_simple_html_string(self, options):
             for field, datum in zip(self._field_names, row):
                 if options["fields"] and field not in options["fields"]:
                     continue
-                lines.append(
-                    "            <td>{}</td>".format(
-                        escape(datum).replace("\n", linebreak)
+                if options["escape_data"]:
+                    lines.append(
+                        "            <td>{}</td>".format(
+                            escape(datum).replace("\n", linebreak)
+                        )
+                    )
+                else:
+                    lines.append(
+                        "            <td>{}</td>".format(datum.replace("\n", linebreak))
                     )
```

Comment:
```suggestion

                if options["escape_data"]:
                    datum = escape(datum)

                lines.append(
                    "            <td>{}</td>".format(datum.replace("\n", linebreak))
                )
```

Same thing about the tests.